test/blackbox/common/RTPSBlackboxTestsBasic.cpp

```diff
@@ -25,6 +25,7 @@
 #include <fastdds/rtps/interfaces/IReaderDataFilter.hpp>
 #include <fastdds/rtps/participant/RTPSParticipant.h>
 #include <fastdds/rtps/RTPSDomain.h>
+#include <fastdds/rtps/transport/shared_mem/SharedMemTransportDescriptor.h>
```

Comment:
We are not using this:

```suggestion
```

Reply:
Yes, I added it at the beginning and then I forgot to remove it